src/api/cpp/cvc5_checks.h

```diff
@@ -155,23 +155,23 @@ class CVC5ApiUnsupportedExceptionStream
 /** Check it 'this' is not a null object. */
 #define CVC5_API_CHECK_NOT_NULL                     \
   CVC5_API_CHECK(!isNullHelper())                   \
-      << "Invalid call to '" << __PRETTY_FUNCTION__ \
+      << "invalid call to '" << __PRETTY_FUNCTION__ \
       << "', expected non-null object";
 
 /** Check if given argument is not a null object. */
 #define CVC5_API_ARG_CHECK_NOT_NULL(arg) \
-  CVC5_API_CHECK(!arg.isNull()) << "Invalid null argument for '" << #arg << "'";
```

Comment:
I think the trailing ; should be removed again, otherwise we'll end up having ;; when using the macro as MACRO(); This applies to all macros.